src/storage/oid.h

```diff
@@ -92,6 +92,7 @@
     (oidp)->volid = NULL_VOLID; \
   } while(0)
 
+#define oid_is_equal_value(oid1, oid2)  OID_EQ((oid1), (oid2))
```

Comment:
Do we need this macro?

Reply:
I can't say it's absolutely necessary.
However, I tried to make the macro name more explicit while defining a macro that replaces the existing code of the form oid_cpmpare() == 0.

Reply:
If it only calls OID_EQ, this macro is redundant. Also, the name of C macro function should be capital by naming convention.

Reply:
@hgryoo OK, I will fix it.